### PR TITLE
[1LP][RFR] Smoke tests for Control

### PR DESCRIPTION
**Pull request by quarckster**
Intent
=====

__Adding smoke tests__ for Control tab.

{{py.test: -v -k 'test_control_navigation or test_control_explorer_tree'}}

### CODE REVIEW

File: cfme/control/simulation.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    reset_button = Button("Reset")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.event_selection.is_displayed and
+            self.vm_selection.is_displayed and
+            self.submit_button.is_displayed and
+            self.reset_button.is_displayed
+        )
```
Comment (mshriver) [inline cfme/control/simulation.py]: At least a TODO here for the results pane

File: cfme/tests/control/test_smoke_control.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+
+@pytest.mark.parametrize("destination", destinations)
+def test_control_navigation(destination):
+    """This test verifies presence of destinations of Control tab.
+
+    Steps:
+        * Open each destination of Control tab.
+    """
+    navigate_to(Server, destination)
```
Comment (mshriver) [inline cfme/tests/control/test_smoke_control.py]: **Optional**
Capture the view and add an assert against view.is_displayed for an explicit 'test' here.

File: cfme/control/simulation.py (hunk 1 above)
Comment (quarckster) [inline cfme/control/simulation.py]: There is no results pane when you open Simulation tab, the tree with simulation results will be shown only after filling that widgets.

Comment (quarckster) [inline cfme/control/simulation.py]: Done

**Pull request merged**